presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -8000,4 +8000,44 @@ public void testDefaultDecimalLiteralSwitch()
         assertEquals(doubleColumnResult.getTypes().get(0), DOUBLE);
         assertEquals(doubleColumnResult.getMaterializedRows().get(0).getField(0), 1.0);
     }
+
+    @Test
+    public void testInnerJoinShortCircuitOnEmptyBuildTable()
```

Comment:
This test doesn't test that inner join did short-circuit. Maybe call it like `testInnerJoinWithEmptyBuildSide`?